ops/platform/02-backup/main.tf

```diff
@@ -0,0 +1,55 @@
+data "aws_backup_vault" "main" {
+  name = "CMS_OIT_Backups_Vault"
+}
+
+data "aws_backup_vault" "dr" {
+  provider = aws.secondary
+  name     = "CMS_OIT_Backups_Vault"
+}
+
+data "aws_iam_role" "backup" {
+  name = "cms-oit-aws-backup-service-role"
+}
+
+module "terraservice" {
+  source = "../../terraform-modules/bfd/bfd-platform-service"
+
+  greenfield           = var.greenfield
+  service              = local.service
+  relative_module_root = "ops/platform/02-backup"
+}
+
+resource "aws_backup_plan" "main" {
+  name = "bfd-weekly"
+
+  rule {
+    rule_name         = "weekly35"
+    target_vault_name = data.aws_backup_vault.main.name
+    schedule          = "cron(0 3 ? * SAT *)"
+
+    lifecycle {
+      delete_after = 35
+    }
+
+    copy_action {
+      destination_vault_arn = data.aws_backup_vault.dr.arn
+      lifecycle {
+        delete_after = 7
```

Comment:
7 days seems short is that expected?

Reply:
The retention policy for our Snapshots and decided on 3 dailies, 5 weeklies, and the latest weekly being replicated to DR